src/offspot_demo/toggle.py

```diff
@@ -71,3 +74,37 @@ def get_mode() -> Mode:
         if DOCKER_COMPOSE_SYMLINK_PATH.resolve() == DOCKER_COMPOSE_IMAGE_PATH
         else Mode.MAINT
     )
+
+
+def entrypoint():
+    parser = argparse.ArgumentParser(
+        prog="demo-toggle", description="Toggle between maint and image modes"
+    )
+
+    parser.add_argument(
+        dest="mode",
+        help="New target mode, either maint or image",
+        default="maint",
+    )
+
+    args = parser.parse_args()
+    logger.setLevel(logging.DEBUG)
+
+    try:
+        if args.mode == "image":
```

Comment:
```suggestion
        try:
            mode = Mode[args.mode.upper()]
         except KeyError:
             raise ValueError(f"Unsupported mode: {args.mode}, must be either image or maint")
```

You can also/alternatively add a `choices=Mode.__members__.keys()` in the add_argument decl so that argparse validates it